src/wp-content/themes/twentytwentytwo/inc/block-patterns.php

```diff
@@ -132,4 +133,4 @@ function twentytwentytwo_register_block_patterns() {
 		);
 	}
 }
-add_action( 'init', 'twentytwentytwo_register_block_patterns', 9 );
+add_action( 'init', 'twentytwentytwo_register_block_patterns', 21 );
```

Comment:
This is how I got the check for the new comments block to work on my setup running 6.0 + the plugin.

Reply:
Great find! 🕵️‍♀️ Though I'm a bit hesitant to change the priority of this action; there might be a reason it was set to `9` (which I think is one before the default `10`) 🤔

But we could maybe add the `comments` pattern in a separate `action` that we add with priority `21` (just to play it safe) 🤔